.yamllint

```diff
@@ -4,3 +4,4 @@ extends: default
 rules:
   line-length: disable
   trailing-spaces: disable
+  empty-lines: disable
```

Comment:
nit: why remove that? most editors will remove spaces from new lines anyway.

Reply:
to make our lives easier? :)